packages/shared-components/src/components/onto-dropdown/onto-dropdown.tsx

```diff
@@ -37,13 +37,18 @@ export class OntoDropdown {
   /**
    * The dropdown button tooltip. It will be used if present; otherwise, the {@link OntoDropdown#dropdownButtonTooltipLabelKey} will be used.
    */
-  @Prop() dropdownButtonTooltip: string;
+  @Prop() dropdownButtonTooltip: string | (() => Promise<string>);
```

Comment:
we should think for some custom type for these `() => Promise<...>`

Reply:
Done